.ci/install.sh

```diff
@@ -44,7 +44,10 @@ if [[ $TRAVIS_PYTHON_VERSION == 3.* ]]; then
 fi
 
 # docs only on Python 3.8
-if [ "$TRAVIS_PYTHON_VERSION" == "3.8" ]; then pip install -r requirements.txt ; fi
+if [ "$TRAVIS_PYTHON_VERSION" == "3.8" ]; then
+    sudo chown $USER ~/.cache/pip/wheels
```

Comment:
It's coincidental s390x is using Python 3.8.

In case we shuffle the CPUs/Python versions, could we do this in its own `TRAVIS_CPU_ARCH` if block?

Reply:
Ok. I've separated it out and moved it earlier.